packages/utilities/psammead-storybook-helpers/index.test.jsx

```diff
@@ -0,0 +1,44 @@
+import { testUtilityPackages } from '@bbc/psammead-test-helpers';
+import * as index from './dist/index';
```

Comment:
Now that there isnt a base level JS file, and your using dist as the `main`, you dont need to test both src and dist